docs/index.rst

```diff
@@ -8,20 +8,36 @@ implementations using the reactive pattern.
 Overview
 --------
 
-The pattern is "reactive" because you use :func:`@when <charms.reactive.decorators.when>`
-and similar decorators to indicate that blocks of code react to certain conditions,
-such as a relation reaching a specific `state`, a file changing, certain config
-values being set, etc.  More importantly, you can react to not just individual
-conditions, but meaningful combinations of conditions that can span multiple hook
-invocations, in a natural way.
+Juju is an open source tool for modelling a connected set of applications in a
+way that allows for that model to be deployed repeatably and consistently
+across different clouds and substrates.  Juju Charms model the individual
```

Comment:
Not sure if model is the correct term here. Maybe they 'implement the model for individual applications' rather than 'model the individual applications'.